internal/service/device.go

```diff
@@ -22,6 +23,8 @@ type DeviceStoreInterface interface {
 	UpdateDeviceStatus(ctx context.Context, orgId uuid.UUID, device *api.Device) (*api.Device, error)
 	DeleteDevices(ctx context.Context, orgId uuid.UUID) error
 	DeleteDevice(ctx context.Context, orgId uuid.UUID, name string) error
+	ListAllDevicesInternal(map[string]string) ([]model.Device, error)
```

Comment:
nit: in general it feels the interface naming studders since we using a `DeviceStore` and naming it `Interface` is unnecessary. It also feels awkward to have an internal naming designation on a public-facing method of an interface. These things will involve rework in the future and it would be nice to break these patterns?

```
type DeviceStore interface {
  ....
  UpdateStatus
  DeleteAll
  // Get all Devices regardless of ownership. Used internally by the DeviceUpdater.
  ListAll
```

Reply:
I agree.  Want to submit a PR? :)
By "Internal", I mean that it's not used for handling API calls, but rather for calls from within the service.  The main difference is the lack of orgId.

Reply:
Yeah not pointing fingers here of course :) but yeah I will take some time to outline an alternative path forward in a PR down the road hopefully soon and we can talk about it.

Reply:
Great, thanks!